.github/workflows/prepare-release.yml

```diff
@@ -164,24 +170,22 @@ jobs:
       - name: Build native test server (non-Docker)
         if: matrix.os_family != 'Linux'
         run: |
-          ./gradlew :temporal-test-server:build
+          ./gradlew -PnativeBuild :temporal-test-server:nativeBuild
 
       - name: Build native test server (Docker)
         if: matrix.os_family == 'Linux'
         run: |
           docker run \
             --rm -w /github/workspace -v "$(pwd):/github/workspace" \
             $(docker build -q ./docker/native-image) \
-            sh -c "./gradlew :temporal-test-server:build"
+            sh -c "./gradlew -PnativeBuild :temporal-test-server:nativeBuild"
       # path ends in a wildcard because on windows the file ends in '.exe'
-      # path excludes *.txt because native-image also writes a build manifest txt file
       - name: Upload executable to workflow
         uses: actions/upload-artifact@v4
         with:
           name: ${{ matrix.os_family }}_${{ matrix.arch }}
           path: |
-            temporal-test-server/build/graal/temporal-test-server*
-            !temporal-test-server/build/graal/*.txt
+            temporal-test-server/build/native/nativeCompile/temporal-test-server*
```

Comment:
Can I get confirmation that we believe the release artifact files will have the same name, just with arm ones now? So `temporal-test-server_<version>_linux_arm64.tar.gz` and `temporal-test-server_<version>_maxOS_arm64.tar.gz` will now be present?

Reply:
Yep